File: 백준/Bronze/11720. 숫자의 합/숫자의 합.c
#include<stdio.h>

int main(){
	char a[10000];
	int num=0;
	int sum=0;
	scanf("%d",&num);
	scanf("%s",a);
	for(int i=0; i<num; i++){	
		
    a[i]=(int)a[i];
		a[i]=a[i]-48;
		sum+=a[i];
	}
	printf("%d",sum);
}

File: 백준/Bronze/11720. 숫자의 합/숫자의 합.js
let input = require('fs').readFileSync('/dev/stdin').toString().trim().split('\n');

let num = input[1];

const add = (array) => {
    array = array.map((item) => Number(item));
    const sum = array.reduce((sum, cur) => sum + cur);
    console.log(sum);
}

const solution = () => {
    num = num.split('');
    add(num);
}

solution();